src/alfasim_sdk/_internal/types.py

```diff
@@ -210,9 +210,10 @@ class String(BaseField):
     """
     The String field represents an input that allows the user to enter and edit a single line of plain text.
 
-    The String field have all options available from :func:`~alfasim_sdk.types.BaseField`, plus the following ones
+    The String field have all options available from ```BaseField```, plus the following ones
```

Comment:
Yeah, it can't find the reference for BaseField. `WARNING: py:class reference target not found: BaseField`

I was searching about it, and probably we need to change the `99_2_user_interface_reference` to use `.. automodule::`

Reply:
For me it is declared in `alfasim_sdk._internal.types`, so I suppose it is just a matter of using

```
:func:`~alfasim_sdk._internal.types.BaseField`
```

?

Reply:
I tried it, but seems that sphinx can't refer to the base class this way.

Reply:
Hey, find out that the problem was that in `99_2_user_interface_reference.rst` BaseField was using the full path in `.. autoclass:: alfasim_sdk._internal.types.BaseField()`and the other classes are using the short version `.. autoclass:: alfasim_sdk.String()` (they are imported in __init__). Importing `BaseField`and using the short path solves the problem, now we can reference the base class.